packages/ember-routing/lib/services/routing.js

```diff
@@ -48,13 +48,16 @@ export default Service.extend({
   },
 
   generateURL(routeName, models, queryParams) {
+    let router = get(this, 'router');
+    if (!router._routerMicrolib) { return; }
```

Comment:
Can you add a comment above this to explain why its needed?

Reply:
done